src/BaGet/Startup.cs

```diff
@@ -5,8 +5,8 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
Can we revert this change? This is no longer sorted alphabetically

Reply:
Yep.